drivers/display/display_gc9x01x.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2023 Mr Beam Lasers GmbH.
+ * Copyright (c) 2023 Amrith Venkat Kesavamoorthi <amrith@mr-beam.org>
+ * Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_DRIVERS_DISPLAY_GC9X01X_H_
+#define ZEPHYR_DRIVERS_DISPLAY_GC9X01X_H_
+
+#include <zephyr/sys/util.h>
+
+/* Command registers */
+#define GC9X01X_CMD_SLPIN     (0x10U) /* Enter Sleep Mode */
+#define GC9X01X_CMD_SLPOUT    (0x11U) /* Exit Sleep Mode */
+#define GC9X01X_CMD_PTLON     (0x12U) /* Partial Mode ON */
```

Comment:
Some of macros are not used. Are they for documentation purpose here?

Reply:
Yes, it's for documentation and completeness purpose. I added definitions for all registers from the datasheet. If someone else is going to extend functionality he can make use of this. These defines do not consume any memory, so I think that should be fine.